marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/data-hub-operator.json

```diff
@@ -5,7 +5,13 @@
     "rest-extension-user",
     "rest-reader",
     "rest-writer",
-    "data-hub-job-reader"
+    "data-hub-job-reader",
```

Comment:
The operator should only have reader roles

Reply:
Changed the roles so that operator has reader roles and developer has writer roles